processor/processor.go

```diff
@@ -797,6 +798,106 @@ func (proc *HandleT) updateSourceEventStatsDetailed(event types.SingularEventT,
 		}
 	}
 }
+
+type TransformationStatusT struct {
+	sourceID          string
+	destID            string
+	destination       *backendconfig.DestinationT
+	eventsToTransform []transformer.TransformerEventT
```

Comment:
change eventsToTranform name to something like userTransformedEvents for readability?